### DISCUSSION TITLE
I might be stupid but I can't get it to work on macOS 13

### DISCUSSION THREAD

**Discussion by rwidmark** [JanDeDobbeleer/oh-my-posh#3149] [Troubleshoot]
Hi,
Is it any walkthrough step by step? I have been trying with your documentation but I can't seem to get it to work with powershell on macOS or In vscode.
If I get the colors the font are not there and I can't seem to get the font to work I'm trying with the one that you recommended in the documentation.

**Comment by JanDeDobbeleer** [comment]
@rstolpe there are a few steps you need to do for fonts to work:

- install the font on your Mac (you can `oh-my-posh font install` for this)
- restart the terminal and code
- configure both to use that font (code needs that for the integrated terminal setting)

**Reply by rwidmark** [reply]
I have tried everything that I can think of it's the same in terminal, PowerShell and vscode.
Here is my steps that I have done.
1. Installed it with brew ```brew install jandedobbeleer/oh-my-posh/oh-my-posh```
2. Installed the fonts ```oh-my-posh font install```
As I wanted Meslo LGM NF I did select Meslo in the list that did show when I did run the font install. Everything seems OK it did say downloading and it did return "Successfully installed Meslo 🚀"
3. In the ```~/.zshrc``` file I did add
````
eval "$(oh-my-posh init zsh)"
````
4. I did also open settings for terminal and changed font to "MesloLGS NFM"
![image](https://user-images.githubusercontent.com/76907327/205000774-e9127552-696d-4265-a681-ee729c80868c.png)

5. In PowerShell I did add the following to my profile ($profile) path
```oh-my-posh init pwsh --config 'https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/jandedobbeleer.omp.json' | Invoke-Expression```
I did add that both on ```Microsoft.VSCode_profile``` and inside my ```Microsoft.PowerShell_profile.ps1```
6. In vsCode I do open the settings.json file and adding the following.
````
    "terminal.integrated.fontFamily": "MesloLGS NFM"
````

So it seem that I can't add the path or similar to the fonts correctly for PowerShell and vsCode.

I do get it to work in iTerm2 but that's it sadly.
And there I can run both terminal and powershell so that's OK, but that I can't get it to work in vsCode is strange :/

and here is how it looks like in vscode:
![image](https://user-images.githubusercontent.com/76907327/204997169-89fdf81c-da9e-46fa-b768-79e934d7a1a8.png)

**Reply by JanDeDobbeleer** [reply]
@rstolpe the font name in code is wrong. You'll need to have a look in font book to get the full font name (which could be the same as what iTerm displays).

**Reply by rwidmark** [reply]
Thanks! Don't know why I thought that was the correct name of the font.
